syntax = "proto3";

package user;

message UserRequest {
    string user_id = 1;
}

message UserResponse {
    string user_id = 1;
    string username = 2;
    string email = 3;
    int32 age = 4;
}

service UserService {
    rpc GetUser(UserRequest) returns (UserResponse);
}
